service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -745,7 +745,7 @@ EOF
 if [[ -n "${INSTANCE_CONTAINER}" ]]; then
 
 # Indicate the end of Terra customizations of the jupyter_notebook_config.py
```

Comment:
Please move this below your changes such that we have the customizations end markers. Also remove the "if [[ -n "${INSTANCE_CONTAINER}" ]]; then" wrapper since we now always write to the NOTEBOOK_CONFIG file.